pkg/apis/velero/v1/delete_backup_request.go

```diff
@@ -52,6 +52,9 @@ type DeleteBackupRequestStatus struct {
 
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:printcolumn:name="Status",type="string",JSONPath=".status.phase",description="Delete Backup Request status such as InProgress/Processed"
+// +kubebuilder:printcolumn:name="Backup Name",type="string",JSONPath=".spec.backupName",description="Name of backup to be deleted"
+// +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp"
```

Comment:
suggest: `Created At`

Reply:
We should stick with `Age` here and in the BSL, or change both to `Created At`. `Age` sounds good to me, especially since a lot/most/(all) k8s resources have this column.

Reply:
It looks like `Age` on built-in kubernetes resources is a calculated field, not simply a timestamp.

So if we're not calculating the age here, I don't think we should re-use that name, because that's actually using the same name for something different but similar.

As an example, here's what nodes report:

```
% k get nodes
NAME                 STATUS   ROLES    AGE   VERSION
kind-control-plane   Ready    master   19h   v1.17.0
```

Reply:
Kubebuilder does the age calculation when you name a column "Age". BSL uses the same marker, you may verify it with that.

Reply:
Yes, kubebuilder seems to handle it correctly. For all the `Age` columns that I added, the output is a relative time, for example:

```
k get deletebackuprequests.velero.io -n velero
NAME                      STATUS       BACKUP NAME         AGE
test-pod-backup-2-pc969   InProgress   test-pod-backup-2   16d
test-pod-backup-3-h4689   InProgress   test-pod-backup-3   38s
```

Reply:
This doesn't seem to be specific to the `Age` column though. It seems anytime that the `date` type is used, this is how it is displayed by kubectl. For example:

```
 $ k get restores.velero.io -n velero
NAME                               BACKUP              STATUS            STARTED   COMPLETED   ERRORS   WARNINGS   AGE
test-pod-backup-20200831105635     test-pod-backup     Completed         17d       17d                  27         17d
test-pod-backup-3-20200831122428   test-pod-backup-3   Completed         17d       17d                  27         17d
test-pod-backup-3-20200831122623   test-pod-backup-3   Completed         17d       17d                  26         17d
test-pod-backup-3-manul            test-pod-backup-3   PartiallyFailed   15d       15d         7        22         15d
```

All timestamp values are displayed as a duration from the current time.

Reply:
Ah yeah, the name of the column doesn't matter, it is the (date) type!